### PR TITLE
Fixes #9491: Broken policy update

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9491

### CODE REVIEW

**Comment by amousset** [comment]
Tested successfully.

**Comment by amousset** [comment]
This also removes almost 400 lines from failsafe!

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -839,6 +839,85 @@ bundle agent check_binaries_freshness
 }
 
 #######################################################
+# This bundle is responsible for the reporting of what happened in the update
+# It can work because the classes defined during the update are persistent, so
+# the classes are available for the next 4 minutes
+bundle agent update_reports
```
Comment (ncharles) [inline techniques/system/common/1.0/promises.st]: this is in the wrong file

Comment (ncharles) [inline techniques/system/common/1.0/promises.st]: failsafe won't see it

Comment (ncharles) [inline techniques/system/common/1.0/promises.st]: ha, sorryn misread

Comment (peckpeck) [inline techniques/system/common/1.0/promises.st]: failsafe never calls it

**Pull request merged**